packages/php/email-editor/src/Integrations/Core/Renderer/Blocks/class-quote.php

```diff
@@ -0,0 +1,134 @@
+	 * @return string The wrapped citation HTML or empty string if no citation.
+	 */
+	private function get_citation_wrapper( string $citation_content, array $parsed_block ): string {
+		if ( empty( $citation_content ) ) {
+			return '';
+		}
+
+		return $this->add_spacer(
+			sprintf(
+				'<p style="margin: 0; %2$s"><cite class="email-block-quote-citation" style="display: block; margin: 0;">%1$s</cite></p>',
+				$citation_content,
+				WP_Style_Engine::compile_css( array( 'text-align' => $parsed_block['attrs']['textAlign'] ?? '' ), '' ),
+			),
+			$parsed_block['email_attrs'] ?? array()
+		);
+	}
+
+	/**
+	 * Returns the block wrapper.
+	 *
+	 * @param string              $block_content Block content.
+	 * @param array               $parsed_block Parsed block.
+	 * @param Settings_Controller $settings_controller Settings controller.
+	 */
+	private function get_block_wrapper( string $block_content, array $parsed_block, Settings_Controller $settings_controller ): string {
+		$original_classname = ( new Dom_Document_Helper( $block_content ) )->get_attribute_value_by_tag_name( 'blockquote', 'class' ) ?? '';
+		$block_attributes   = wp_parse_args(
+			$parsed_block['attrs'] ?? array(),
+			array(
+				'style'           => array(),
+				'backgroundColor' => '',
+				'textColor'       => '',
+			)
+		);
+
+		// Layout, background, borders need to be on the outer table element.
+		$table_styles = $this->get_styles_from_block(
+			array(
+				'color'      => array_filter(
+					array(
+						'background' => $block_attributes['backgroundColor'] ? $settings_controller->translate_slug_to_color( $block_attributes['backgroundColor'] ) : null,
+						'text'       => $block_attributes['textColor'] ? $settings_controller->translate_slug_to_color( $block_attributes['textColor'] ) : null,
+					)
+				),
+				'background' => $block_attributes['style']['background'] ?? array(),
+				'border'     => $block_attributes['style']['border'] ?? array(),
+			)
+		)['declarations'];
+
+		// Set the text align attribute to the wrapper if present.
+		if ( isset( $parsed_block['attrs']['textAlign'] ) ) {
+			$table_styles['text-align'] = $parsed_block['attrs']['textAlign'];
+		}
+
+		$table_styles['border-collapse'] = 'separate'; // Needed for the border radius to work.
+
+		// Add default background size.
+		$table_styles['background-size'] = empty( $table_styles['background-size'] ) ? 'cover' : $table_styles['background-size'];
+
+		// Padding properties need to be added to the table cell.
+		$cell_styles = $this->get_styles_from_block(
+			array(
+				'spacing' => array( 'padding' => $block_attributes['style']['spacing']['padding'] ?? array() ),
+			)
+		)['declarations'];
+
+		return sprintf(
+			'<!--[if mso | IE]><table align="left" role="presentation" border="0" cellpadding="0" cellspacing="0" width="100%%" style="%1$s"><tr><td style="%2$s" width="100%%"><![endif]-->
+			<blockquote class="email-block-quote %3$s" style="%1$s %2$s">
```

Comment:
Thank you for adding the blockquote tag here. Unfortunately, when I tested the latest changes, I had some issues with the Gmail mobile app on EmailOnAcid. It looks like styling this HTML tag is not possible, and the left and right borders are missing. I am afraid we need to avoid using this tag because of the Gmail app.
<img width="398" alt="Screenshot 2025-05-06 at 18 51 56" src="https://github.com/user-attachments/assets/ea7738f7-61ae-4046-8c30-14c05145b049" />

Reply:
Apologies - I didn't realize it was broken. I updated the email source code using the `<blockquote>` tag on Email On Acid, but I was apparently looking at outdated screenshots and thought everything was working fine.

It turns out the `<blockquote>` tag also caused the border style issue you mentioned in your other comment. Reverting back to using tables seems to have resolved all of these issues.

Do you think we should also replace the `<cite>` tag with a table? From my testing, the `<cite>` tag appears to render well across email clients, but I wanted to double check.